_im-plugin/ism/api-config-notice.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: default
+title: Configuring ISM notifications settings
+parent: ISM API
+grand_parent: Index State Management
+nav_order: 10
+---
+
+## Configuring ISM notifications settings
+
+Introduced 2.8
+{: .label .label-purple }
+
+Notifications settings are configured globally, so they do not need to be configured manually for each operation. Notifications settings allow users to control how they receive notifications about index events. For example, you can choose to receive email notifications, Slack channel notifications, or both. This documentation is intended for IT administrators as a guide to configuring notifications settings.
+
+The following notification types are supported: 
```

Comment:
All notification types in notifications plugin are supported. But only a few index operations are supported: resize/reindex/open index/force merge.